samples/subsys/usb/hid/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_USB=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_HID=y
+
+CONFIG_SYS_LOG=y
+CONFIG_SYS_LOG_USB_DRIVER_LEVEL=1
+CONFIG_SYS_LOG_USB_DEVICE_LEVEL=1
```

Comment:
Would you like to add CONFIG_SYS_LOG_USB_HID_LEVEL=1 here?

Reply:
I do not think we need a level specific for HID, USB_DEVICE should be fine for all USB devices.

Reply:
It is then not consistent with existing code, log level can be set separately for each class. It is quite useful when debugging composite usb device.

Reply:
yes, I noticed some classes have there own, but that was already inconsistent, for example DFU does not have its own Kconfig. IMO we should reduce this to just one USB DEVICE logging layer and get rid of the too many Kconfigs, it becomes very difficult to get this right and getting logging to work correctly requires lots of kconfigs. We had this for sensors before, where every sensor had its own log level, then move to one top level Kconfig for all sensors.

Reply:
btw, there will be a major rework of syslog coming soon with different backends and based on some proposal, a different way for managing log domains.

Reply:
@nashif general USB DEVICE log may print too much logging, which is for USB means breaking timing and it is better to leave those logging configurations. I do not have it for HID since it is much simpler then networking and we handle less data.

Reply:
thats why logging will have different backends and you will not have everything console bound, i.e. you will be able to log into memory and retrieve logs and filter them if needed. Having a logging layer on every file is really excessive. You might also end up missing lots of useful information from other related domains if the only thing you show is coming from one file. anyways, this is a discussion for another PR/Issue.